embedchain/chunkers/base_chunker.py

```diff
@@ -23,6 +26,8 @@ def create_chunks(self, loader, src, app_id=None):
         documents = []
         chunk_ids = []
         idMap = {}
+        min_chunk_size = config.min_chunk_size if config is not None else 0
```

Comment:
Should we set the default to 1 instead of 0 since `min_chunk_size` as 0 doesn't make any sense?

Reply:
okay, sounds good!